### ISSUE TITLE
Clarify README: EventSourceMapping

### ISSUE THREAD

**Issue by batpad** [mapbox/streambot#46]
As I understand it, we no longer need to use the custom EventSourceMappings since AWS now supports this natively in CloudFormation.

@rclark can we remove instructions to add the custom EventSourceMapping function from the README?

**Comment by rclark** [comment]
Yes, we can and should remove it from the readme. However we can't remove it from the repo entirely just yet.
